src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp

```diff
@@ -252,6 +252,29 @@ void RendererSettings::apply_common_settings(asr::Project& project, const char*
     if (m_max_ray_intensity_set)
         params.insert_path("pt.max_ray_intensity", m_max_ray_intensity);
 
+    params.insert_path("lighting_engine", get_lighting_engine_type(m_lighting_algorithm));
```

Comment:
Move up (right after setting the sampling mode, for instance); right now it's in the middle of PT-specific and SPPM-specific settings.